keps/sig-auth/1513-certificate-signing-request/README.md

```diff
@@ -323,6 +328,11 @@ type CertificateSigningRequestSpec struct {
   // You can select on this field using `.spec.signerName`.
   SignerName string
 
+  // durationHint is a hint to the signer in regards to when the issued certificate should expire.
+  // The signer may or may not honor this field.  The well-known kubernetes signers will honor this field
+  // as long as the requested duration is not later than the maximum duration they will honor.
```

Comment:
I'd also expect a minimum duration, similar to the minimum token lifetime we grant. Describe the behavior of the build-in signers when the hint is less than the min or greater than the max (clip to the min/max, or revert to the default)?

Reply:
also, since this is an addition, we can't back-fill the hint honoring into the kubernetes signer definitions (since implementations before this field exists won't honor it), but we can say the in-tree implementations of the kubernetes signers will honor it from version X forward)